src/index.ts

```diff
@@ -14,9 +14,14 @@ function getInputAsArray(name: string, options?: core.InputOptions): string[] {
     .filter((x) => x !== '')
 }
 
+function getBooleanInput(name: string, options: core.InputOptions): boolean {
```

Comment:
We use the old version of `@action/core`. It doesn't include the getBooleanInput API. I create one to have it.

Reply:
We should just upgrade the `@action/core` dependency.